cub/cub/util_type.cuh

```diff
@@ -1370,6 +1370,25 @@ template <> struct NumericTraits<bool> :                BaseTraits<UNSIGNED_INTE
 template <typename T>
 struct Traits : NumericTraits<typename ::cuda::std::remove_cv<T>::type> {};
 
+namespace detail 
+{
+
+template <::cuda::std::size_t Alignment, class... T> 
+struct max_alignment_t;
+
+template <::cuda::std::size_t Alignment> 
+struct max_alignment_t<Alignment> 
+{
+  constexpr static ::cuda::std::size_t value = Alignment;
```

Comment:
Suggestion:

we usually go with the ordering of `static constexpr` because static is the more relevant information here

Reply:
Remark: clang-format can handle such orderings using the [QualifierOrder](https://clang.llvm.org/docs/ClangFormatStyleOptions.html#qualifierorder) style option, which we seem to not set in our `.clang-format`.

Reply:
Opened: #1748